apps/dashboard/src/routes/org/[slug]/community/ask/+page.svelte

```diff
@@ -49,6 +63,10 @@
       goto(`${$currentOrgPath}/community/${slug}`);
     }
   }
+
+  onMount(() => {
+    getCourses($profile.id, $currentOrg.id);
+  });
```

Comment:
Follow the review I gave here: apps/dashboard/src/routes/lms/community/ask/+page.svelte